charts/exporter/values.yaml

```
@@ -0,0 +1,41 @@
+annotations: {}
+replicaCount: 1
+imagePullPolicy: Always
+imagePullSecrets: []
+nodeSelector: {}
+tolerations: []
+affinity: {}
+securityContext: {}
+  # fsGroup: 1001
+  # runAsGroup: 1001
+  # runAsNonRoot: true
+  # runAsUser: 1001
```

Why are these (and the resources below) commented out? As an example?

---

As example.
I didn't dive deep into `securityContext`, for now.
Perhaps you'll advise better default value for the parameter?

---

```suggestion
    fsGroup: 100
    runAsGroup: 100
    runAsNonRoot: true
    runAsUser: 100
```

The image runs as user `100` by default, so let's go with that.